Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/NavigationDrawerFragment.kt

```diff
@@ -342,7 +342,7 @@ class NavigationDrawerFragment : DialogFragment() {
         val specialItems = user.items?.special
         var hasSpecialItems = false
         if (specialItems != null) {
-            hasSpecialItems = specialItems.hasSpecialItems
+            hasSpecialItems = true
```

Comment:
This would show the skills menu item in some cases where we don't want it. I think instead the `Items` class should have a method to check if the user has transformation items (with the same logic from the previous `hasSpecialItems` method. To clarify the usage that should then be called `hasTransformationItems` instead, since it wouldn't count cards or mystery gifts